src/scripts/main.js

```diff
@@ -1,3 +1,23 @@
 'use strict';
 
 // write code here
+
+const items = document.querySelector('li');
```

Comment:
The `items` variable is assigned a single `li` element. It should be `document.querySelectorAll('li')` to select all `li` elements as a NodeList for iteration.